src/main/java/pl/wrapper/parking/facade/domain/ParkingController.java

```diff
@@ -1,13 +1,34 @@
 package pl.wrapper.parking.facade.domain;
 
+import jakarta.servlet.http.HttpServletRequest;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import pl.wrapper.parking.facade.ParkingService;
 
-@RequestMapping("v1")
+import static pl.wrapper.parking.infrastructure.error.HandleResult.*;
+
+@RequestMapping("/v1/parkings")
 @RestController
 @RequiredArgsConstructor
+@Slf4j
 class ParkingController {
```

Comment:
Urls are clashing with those on develop. Let's change them. The getAll method can be removed

Reply:
I think that getAll method is useful to display all available parkings